### PR TITLE
Added getter for KafkaStreams in order to get QueryableState

### PR DESCRIPTION
**Pull request by ivanursul**

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@ivanursul Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-kafka&pullRequestId=305)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-kafka&pullRequestId=305) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@ivanursul Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-kafka&pullRequestId=305)!

File: spring-kafka/src/main/java/org/springframework/kafka/core/KStreamBuilderFactoryBean.java [hunk 1]
```diff
@@ -141,4 +141,7 @@ public int getPhase() {
 		return this.phase;
 	}
 
+	public KafkaStreams getKafkaStreams() {
```
Comment (artembilan) [inline spring-kafka/src/main/java/org/springframework/kafka/core/KStreamBuilderFactoryBean.java]: Please, add some JavaDocs on the matter and will merge it.

Thank you!

**Comment by artembilan** [comment]
Merged as https://github.com/spring-projects/spring-kafka/commit/b1036bc7d8083b30dcabeb8455e675ed265edec8 and cherry-picked to `1.2.x` and `1.1.x`.

Thank you for contribution, @ivanursul .
Looking forward for more!

**Comment by ivanursul** [comment]
Any plans for release ?

**Comment by artembilan** [comment]
Well, we have just released `2.0 M1` last week and versions `1.1.4` and `1.2.0` almost month ago.
The next `2.0 M2` is planned on the arbitrary day on May 18, `1.1.5` on May 24.
I don't see plans for the `1.2.1`, but any way we can be flexible if there is a strong requirement.

@garyrussell ,

your thoughts?

**Comment by garyrussell** [comment]
I added a 1.2.1 milestone - @ivanursul we can do a release at any time; there have been a few bug fixes since the last releases.

When do you need it?

**Comment by ivanursul** [comment]
@garyrussell would be good to have it as soon as possible.

**Comment by artembilan** [comment]
>  to have it as soon as possible.

@ivanursul ,

which version, please?

**Comment by ivanursul** [comment]
I'm using 1.2.0.RELEASE
Would be good to have 1.2.1

**Comment by artembilan** [comment]
Good. Right now it is the same May 24.
How much time can you afford to wait if not that day?

**Comment by ivanursul** [comment]
Is it possible to have it by the end of this week ?

**Comment by garyrussell** [comment]
We'll release tomorrow - I've updated the milestones on github.

**Comment by ivanursul** [comment]
Thanks guys - you are very flexible and very fast :)

**Comment by garyrussell** [comment]
@ivanursul Sorry for the delay `1.2.1.RELEASE` is in `repo.spring.io/release` and should be in maven central shortly.

**Pull request closed without merging**